src/smk/tool/location/tool-location.js

```diff
@@ -31,20 +31,20 @@ include.module( 'tool-location', [
             this.geocoder = new SMK.TYPE.Geocoder( this.geocoderService )
 
             this.setIdentifyHandler = function ( handler ) {
-                if ( !smk.$tool.identify ) return
+                // if ( !smk.$tool.identify ) return
 
```

Comment:
I'm not a fan of leaving commented out code in the code base, especially since Git will remember it for us. If you choose to go this route, could you please add comments to each section you are commenting out stating why you are doing it? Also, please consider commenting out code at a 'higher level' as well. For example, you might as well comment out the entire this.setIdentifyHandler function itself and comment out any calls to this function. Ditto for this.setDirectionsHandler.

Reply:
Thanks for your thoughts. I'm persuaded to remove the code instead of commenting it out. It's always more fun to remove unused code too :) . My only concern is being able to find this removed code some later day should we want to reinstate functionality ... much git tooling only shows the most recent change in a file/on a line, so it might be harder to see when it was removed. If you have suggestions, let me know. I think I'll update the commit message's first line to be very clear as to what was removed and why, since the first line is what we'll see in git history.